### ISSUE TITLE
Playback error

### ISSUE THREAD

**Issue by warcris** [25huizengek1/ViTune#553]
### Steps to reproduce the bug

When playing music player always on "An unknown playback error has occurred"

### Expected behavior

Always on every song, song will play again after you click pause them play again to load to stop "An unknown playback error has occurred" error

### Actual behavior

N/A

### Screenshots/Screen recordings

![IMG_20241105_194648](https://github.com/user-attachments/assets/a408daaf-9ec4-45d0-8bcc-027ee7e62bb9)

### Logs

N/A

### ViTune version

1.1.1

### What kind of build are you using?

Release (GitHub / F-Droid)

### Android version

Android 10

### Device info

Redmi note 7

### Upstream reproducibility

N/A

### Additional information

N/A

### Checklist

- [X] I am able to reach YouTube Music through the **official** app/website and can confirm that I live in a [supported country](https://support.google.com/youtubemusic/answer/6313540?hl=en) (and this country is not temporarily unavailable)
- [X] I am not using a proxy, VPN, device policy or anything else that restricts access to YouTube Music
- [X] This issue has not been submitted already (this is not a duplicate of another issue)

**Comment by Wqttzicue** [comment]
Ingek will probably ask you for logs

**Comment by 25huizengek1** [comment]
Yup. @warcris Please provide some logs using Settings --> Other --> Show troubleshoot (scroll all the way down) --> Show debug logs --> Share icon

**Comment by 25huizengek1** [comment]
Closing because of inactivity. Probably better off tracking in #555 because it seems similar. Feel free to reopen though!

**Comment by MobasirSarkar** [comment]
Here is the debug logs 


--------- beginning of main
[2025-03-02T13:41:47.314Z] Info (21182) Choreographer - Skipped 33 frames!  The application may be doing too much work on its main thread.
[2025-03-02T13:41:47.342Z] Info (21182) .vitune.android - Compiler allocated 4198KB to compile java.lang.Object A.t.invoke(java.lang.Object, java.lang.Object)
[2025-03-02T13:41:47.616Z] Warning (21182) .vitune.android - Missing inline cache for java.lang.Object A0.t.invoke(java.lang.Object)
[2025-03-02T13:41:47.633Z] Warning (21182) .vitune.android - Missing inline cache for java.lang.Object A0.q.invoke()
[2025-03-02T13:41:48.279Z] Warning (21182) WindowOnBackDispatcher - sendCancelIfRunning: isInProgress=false callback=c.w@9a93161
[2025-03-02T13:42:46.997Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:47.050Z] Warning (21182) WindowOnBackDispatcher - sendCancelIfRunning: isInProgress=false callback=c.w@9a93161
[2025-03-02T13:42:48.630Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:51.710Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:51.986Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:54.693Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:55.456Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:56.382Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:42:57.868Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:00.099Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:03.281Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:03.765Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:04.425Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:04.768Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:05.390Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:06.171Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:06.561Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:11.766Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:12.120Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:12.796Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:19.715Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:20.475Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:22.038Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:22.888Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:25.645Z] Info (21182) ScrollIdentify - on fling
[2025-03-02T13:43:27.370Z] Debug (21182) VRI[MainActivity] - visibilityChanged oldVisibility=true newVisibility=false
[2025-03-02T13:43:27.416Z] Debug (21182) BLASTBufferQueue - [VRI[MainActivity]#1](f:0,a:1) destructor()
[2025-03-02T13:43:27.417Z] Debug (21182) BufferQueueConsumer - [VRI[MainActivity]#1(BLAST Consumer)1](id:52be00000001,api:0,p:-1,c:21182) disconnect
[2025-03-02T13:43:27.418Z] Warning (21182) WindowOnBackDispatcher - sendCancelIfRunning: isInProgress=false callback=c.w@9a93161
[2025-03-02T13:43:30.420Z] Debug (21182) BufferQueueConsumer - [](id:52be00000002,api:0,p:-1,c:21182) connect: controlledByApp=false
[2025-03-02T13:43:30.421Z] Debug (21182) BLASTBufferQueue - [VRI[MainActivity]#2](f:0,a:0) constructor()
[2025-03-02T13:43:30.421Z] Debug (21182) BLASTBufferQueue - [VRI[MainActivity]#2](f:0,a:0) update width=1084 height=2412 format=-1 mTransformHint=0
[2025-03-02T13:43:30.429Z] Debug (21182) BLASTBufferQueue - [VRI[MainActivity]#2](f:0,a:1) acquireNextBufferLocked size=1084x2412 mFrameNumber=1 applyTransaction=true mTimestamp=10282617265688(auto) mPendingTransactions.size=0 graphicBufferId=90975997263895 transform=0
--------- beginning of system
[2025-03-02T13:43:30.451Z] Debug (21182) NtViewRootImpl - mPopUpViewOffsets: offset=(0.0, 0.0), scale=(1.0, 1.0)
[2025-03-02T13:43:32.486Z] Info (21182) ScrollIdentify - on fling
